covid.py

```diff
@@ -1 +1,112 @@
-#this is comment
+import argparse
+import ast
+import json
+import requests
+import os
+
+cache_file = "data.txt"
+
+
+def get_parsed_arguments():
+    my_parser = argparse.ArgumentParser(
+        description="By giving name of country you will get "
+                    "percentage of people recovered and still sick from all population"
+    )
+    my_parser.add_argument(
+        "--country",
+        action="store",
+        required=True,
+        help="Name of country",
+    )
+    return my_parser.parse_args()
+
+
+def compute_answer(population, confirmed_cases, recovered_people, dead_people):
+    percent_of_recovered = round(100 * recovered_people / population, 1)
+
+    percent_of_sick = round(
+        100 * (confirmed_cases - recovered_people - dead_people) / population, 1
+    )
+
+    return json.dumps(
+        {"recovered": percent_of_recovered, "sick": percent_of_sick}, indent=1
+    )
+
+
+def call_covid_api(country_name):
+    response = requests.get(
+        url="https://covid-api.mmediagroup.fr/v1/cases?country=" + country_name
+    )
+    return response
+
+
+def check_is_cache_empty():
+    if os.stat(cache_file).st_size == 0:
+        return True
+    else:
+        return False
+
+
+def save_response_to_cache(response):
+    f = open(cache_file, "w")
+    f.write(str(response.json()))
+    f.close()
+
+
+def get_caches_content():
+    f = open(cache_file, "r")
+    return ast.literal_eval(f.read())
```

Comment:
why don't you use more trival `json.loads(f.read())` ? :thinking:

Reply:
Because as I was "experimenting" with it doesn't seem to work properly. I will check it again, but as I remeber `json.loads()` had problems with quantity of quot signs.

Reply:
![image](https://user-images.githubusercontent.com/76515593/104586357-a97a9200-5665-11eb-9bfa-72e629b9652f.png)

Reply:
it look really weird. We'll revisit it on out call